cmd/monitoring/main.go

```diff
@@ -5,12 +5,18 @@ import (
 
 	"github.com/gagliardetto/solana-go/rpc"
 	relayMonitoring "github.com/smartcontractkit/chainlink-relay/pkg/monitoring"
-	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
 	"github.com/smartcontractkit/chainlink/core/logger"
+
+	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
 )
 
 func main() {
-	coreLog := logger.NewLogger()
+	coreLog, closeLggr := logger.NewLogger()
+	defer func() {
+		if closeLggr != nil {
+			_ = closeLggr()
+		}
+	}()
```

Comment:
I think it is safe to drop the nil check here, and if the error is ignored anyways then could simplify further to just:
```suggestion
	defer closeLggr()
```

Reply:
In this case I get a linter error, that's why I added this check.

Reply:
Hmm ok. Does it accept this?
```suggestion
	defer func() {
		_ = closeLggr()
	}()
```

Reply:
See the comment from @samsondav below, I changed this to log the error, because we should not mute it anyway.